src/vs/workbench/contrib/workspace/browser/workspace.contribution.ts

```diff
@@ -890,3 +908,19 @@ class WorkspaceTrustTelemetryContribution extends Disposable implements IWorkben
 
 Registry.as<IWorkbenchContributionsRegistry>(WorkbenchExtensions.Workbench)
 	.registerWorkbenchContribution(WorkspaceTrustTelemetryContribution, LifecyclePhase.Restored);
+
+function logWorkspaceTrustDialogResult(telemetryService: ITelemetryService, eventName: string, duration: number, result: IShowResult): void {
+	type WorkspaceTrustDialogResultEventClassification = {
+		duration: { classification: 'SystemMetaData', purpose: 'FeatureInsight', isMeasurement: true };
```

Comment:
@lramos15 and myself are suggesting that all new telemetry use expiration. This is a version-based expiration that will stop the event from collection after that version. We can always update the expiry if we still want the data, but this keeps us from over-collection.
https://github.com/microsoft/vscode/blob/fda227db23d2c73912ffce2fcd14ac7c1234a02e/src/vs/platform/telemetry/common/gdprTypings.ts#L8

Reply:
spoken with @lramos15, the format of the string we should use is `1.63` or similar.

Reply:
@isidorn, @sbatten what seems like a reasonable expiration for this type of telemetry?

Reply:
3 milestones seems sufficient to start and decide if we need it for more later

Reply:
I have pushed an update to the PR with an expiration set to `1.64`

Reply:
This expiration works for me, thanks.